aips/aip-x.md

```diff
@@ -0,0 +1,707 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Sign in with Aptos
+author: aptqpv
+Status: Draft # <Draft | Last Call | Accepted | Final | Rejected>
+type: Standard (Ecosystem)
+created: 12/16/2024
+requires: 62
+---
+
+# AIP-X - Sign in with Aptos
+
+## Summary
+
+The "Sign in with Aptos" (SIWA) standard introduces a secure and user-friendly way for users to authenticate to off-chain resources by proving ownership of their blockchain addresses. It simplifies the authentication process by replacing the traditional `connect` + `signMessage` flow in the wallet standard with a streamlined one-click `signIn` method. SIWA leverages decentralized identities to avoid reliance on traditional schemes like SSO while incorporating security measures to combat phishing attacks and improve user visibility.
+
+### Out of scope
+
+The standard relies on public key signature verification. This AIP will not cover specifications or correctness of signature verification.
+
+## High-level Overview
+
+**Decentralized authentication** is a privacy-first solution that empowers individuals to safeguard their information, personalize their experiences, and maintain freedom from censorship.
```

Comment:
I don't think that's what this AIP is about... there are no privacy benefits here over the `connect` + `signIn` baseline, right? 

This AIP is about users signing into a dapp using their Aptos Account and getting access to protected resources.

"Decentralized authentication" sounds like a vague umbrella term to me that should be avoided.